### ISSUE TITLE
Adjust Dynamic Content

### ISSUE THREAD

**Issue by Rana-Fahad-Altaf** [builtbywill/booklet#50]
Is there a possibility to adjust content of html in pages
For Example
I fetch an html from database containing 10000 words and 20 images. I want to make a book where each page has specific height. So I want to split these 10000 words and 20 images in pages.
Is it possible?

Answers would be highly appreciated.

**Comment by tarunshahi** [comment]
Yeah, this is possible you have to be sure about what is the size of image and you can use css property **page-break-after** this will  help you to break the content in next page.I personally advice you to use height and width in rem not in % that will help you in zoom-in and zoom-out .
Hopefully this will help you.
